rts/Lua/LuaUnsyncedCtrl.cpp

```diff
@@ -2769,6 +2771,15 @@ int LuaUnsyncedCtrl::SetDrawSelectionInfo(lua_State* L)
 /******************************************************************************/
 /******************************************************************************/
 
+int LuaUnsyncedCtrl::SetDrawBuild(lua_State* L)
+{
+	if (guihandler != nullptr){
+			guihandler->drawBuildGrid = !!luaL_checkboolean(L, 1);
+			guihandler->drawBuildGhost = !!luaL_checkboolean(L, 2);
+	}
+	return 0;
```

Comment:
Why is this different than the `maintenance` version? I don't see any difference but this will make future cherry-picks harder.
```suggestion
	if (guihandler != nullptr)
		return 0;

	guihandler->drawBuildGrid = luaL_checkboolean(L, 1);
	guihandler->drawBuildGhost = luaL_checkboolean(L, 2);
	return 0;
```
 (Also, no need for the `!!` since `luaL_checkboolean` is already a bool, but this is already not present in the `maintenance` version.)

Reply:
fixing that